indico/modules/events/registration/fields/sessions.py

```diff
@@ -95,7 +95,9 @@ def get_friendly_data(self, registration_data, for_humans=False, for_search=Fals
                   .all())
         if for_humans or for_search:
             return '; '.join(b.full_title for b in blocks)
-        return [_format_block(b, tzinfo) for b in blocks]
+        return [_('{day}: {block}').format(day=format_skeleton(b.start_dt, 'EdM', timezone=tzinfo),
```

Comment:
There's nothing to translate, so you can just leave that out :)
```suggestion
        return ['{day}: {block}'.format(day=format_skeleton(b.start_dt, 'EdM', timezone=tzinfo),
```

Reply:
Sorry then, it was not clear to me about how to use it even after reading the documentation, and I cannot change the language in my core installation to test translations :(

Reply:
Anything that is inside the `_(...)` gets extracted as is and is passed to the translators. Parameters (e.g. `{day}`) do not get translated. The only thing that is actually translatable in this message is the colon, so we can probably just omit the translation altogether and save some time to our translators..

> I cannot change the language in my core installation

Why is that? It's pretty useful when developing features like this to test with different languages

Reply:
I don't know, at UN plugin I have by default English and French, but in core I have only English

Reply:
Maybe you just forgot to compile the translations?

Try running this:
```
indico i18n compile indico
```

Reply:
Yeah, it was that, tyvm! I can see it is translated properly now!